### PR TITLE
Fix nightly CI and update lock

### PR DESCRIPTION
**Pull request by gowthamsk-arm**
The MR has the following changes:
- Fix cargo audit failure
- Update lock file to reflect the toml changes

Signed-off-by: Gowtham Suresh Kumar <gowtham.sureshkumar@arm.com>

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -32,4 +32,4 @@ jobs:
       - name: Install cargo audit
         run: cargo install cargo-audit
       - name: Execute cargo audit
-        run: cargo audit
+        run: cargo audit --ignore RUSTSEC-2021-0139
```
Comment (ionut-arm) [inline .github/workflows/nightly.yml:35]: An alternative to this is to add this ID to the [existing](https://github.com/parallaxsecond/parsec-tool/blob/main/.cargo/audit.toml#L2) list of ignores 😄 Might also be worth checking whether the other ones have been resolved.

Comment (ionut-arm) [reply .github/workflows/nightly.yml:35]: Actually, maybe that's something to include in the release checklist? Don't remember if it's there

Comment (gowthamsk-arm) [reply .github/workflows/nightly.yml:35]: Will add it to the checklist as well. :)

**Review by anta5010 — APPROVED** [review]

**Review by ionut-arm — APPROVED** [review]
🚀

**Pull request merged**